vehicle/tests/golden/compile/andGate/test.json

```diff
@@ -23,6 +23,32 @@
     "external": ["rocq"],
     "ignore": {"files": ["Rocq.v", ".Rocq.aux", "Rocq.glob", "Rocq.vok"]}
   },
+  {
+    "name": "Isabelle",
+    "run": "vehicle compile itp -s spec.vcl -t Isabelle -o isabelle/Isabelle.thy",
+    "needs": [
+      "spec.vcl"
+    ],
+    "produces": [
+      "isabelle/Isabelle.thy"
+    ]
+  },
+  {
+    "name": "IsabelleVerify",
+    "run": "vehicle compile itp -s spec.vcl -t Isabelle -o isabelle/Isabelle.thy && isabelle build -d vehicle-isabelle/ -d isabelle/ IsabelleTest > /dev/null 2>&1 && echo $? || echo 1",
+    "enabled": false,
```

Comment:
These should be enabled, but to stop then running when issabelle is not available, you can add the `external` field (see the Rocq example above).